docker-centos/Dockerfile

```diff
@@ -16,6 +16,10 @@ rpm -V docker-rhel-push-plugin && \
 mkdir -p /exports/hostfs/etc/docker && \
 yum clean all
 
+# Move the /etc/sysconfig/docker out of the way as daemon.json is the proper config location
```

Comment:
@giuseppe We don't, but we do have to do something to the file. It's part of ``docker-common`` and will lay down otherwise.